docs/using-the-jdbc-driver/UsingTheJdbcDriver.md

```diff
@@ -137,7 +137,7 @@ If there is an unreleased feature you would like to try, it may be available in
   <dependency>
     <groupId>software.amazon.jdbc</groupId>
     <artifactId>aws-advanced-jdbc-wrapper</artifactId>
-    <version>2.2.4-SNAPSHOT</version>
+    <version>2.2.5-SNAPSHOT</version>
```

Comment:
shouldn't this be 2.2.6 ?

Reply:
Our latest released version is `2.2.4` (https://github.com/awslabs/aws-advanced-jdbc-wrapper/releases/tag/2.2.4).